src/components/elements/favorites-list.tsx

```diff
@@ -33,53 +37,68 @@ const ShareButtons = () => {
     )
 
   return (
-    <div className="mt-12 flex flex-row justify-between gap-5 *:max-w-60 *:flex-1 *:shrink-0 @md:gap-20">
-      <button
-        className="text-center font-semibold text-archway-dark no-underline hocus:underline"
-        onClick={() => (location.href = smsCopy)}
-        data-course-shared="Favorites"
-        disabled={!favs.length}
-      >
-        <ChatBubbleLeftEllipsisIcon
-          width={30}
-          className={twMerge(
-            "mb-4 block w-full rounded-full bg-spirited-dark p-5 text-white",
-            clsx({"bg-black-40": !favs.length})
-          )}
-        />
-        Text
-      </button>
-      <button
-        className="text-center font-semibold text-archway-dark no-underline hocus:underline"
-        onClick={() => (location.href = emailCopy)}
-        data-course-shared="Favorites"
-        disabled={!favs.length}
-      >
-        <EnvelopeIcon
-          width={30}
-          className={twMerge(
-            "mb-4 block w-full rounded-full bg-spirited-dark p-5 text-white",
-            clsx({"bg-black-40": !favs.length})
-          )}
-        />
-        Email
-      </button>
-      <button
-        className="text-center font-semibold hocus:underline"
-        onClick={() => copyToClip(copyUrl)}
-        data-course-shared="Favorites"
-        disabled={!favs.length}
-      >
-        <ClipboardDocumentIcon
-          width={30}
-          className={twMerge(
-            "mb-4 block w-full rounded-full bg-spirited-dark p-5 text-white",
-            clsx({"bg-black-40": !favs.length})
-          )}
-        />
-        Copy
-      </button>
-    </div>
+    <>
```

Comment:
you could put the toast container inside the div and then this fragment won't be needed.